game/src/Map.cpp

```diff
@@ -0,0 +1,79 @@
+#include <include/Map.hpp>
+
+Map::Map(const uint16_t &id, const uint8_t &currentType, const std::string &mapName, const std::string &bitmapName, const tileArray &tiles, const uint16_t &dimensionX, const uint16_t &dimensionY, const entityMap &entities) {
+    setId(id);
+    setCurrentType(currentType);
+    setMapName(mapName);
+    setBitmapName(bitmapName);
+    setTiles(tiles, dimensionX, dimensionY);
+    setEntities(entities);
+
+}
+
+Map::~Map() {
+    entities_.clear();
```

Comment:
Unnecessary. As long as you don't use the `new` keyboard, the variable takes care cleaning up once it goes out of scope.